setup.py

```diff
@@ -267,7 +267,7 @@ def initialize_options(self):
         self._clean_trees = []
         self._clean_exclude = ['np_datetime.c',
                                'np_datetime_strings.c',
-                               'period.c',
+                               'src/period.c',
```

Comment:
why don't you rename ``period.c`` to say ``period_helper.c`` just to avoid all confusion with the ``.c`` from ``period.pyx``, then yu don't need to use ``src`` prefix here